macos/.gitignore

```diff
@@ -0,0 +1,31 @@
+.idea/
```

Comment:
Why do we need this file? Wouldn't project level enough?

Reply:
Flutter adds automatically when you execute flutter create command to add missing platforms